routes/deductions/deductions.controller.js

```diff
@@ -73,6 +81,10 @@ module.exports = function(app) {
     checkSchema(medicalExpenseSchema),
     checkErrors('deductions/medical'),
     doYesNo('medicalExpenseClaim', 'medicalExpenseAmount'),
```

Comment:
I think if you changed this to "medicalExpenseClaimAmount", it should work out.

Reply:
But the actual pug template might be using a different name than the user.json value.